### DISCUSSION TITLE
Pre-Contribution Due Diligence

### DISCUSSION THREAD

**Discussion by Amini-Community-Auditor-GitHub** [WeAreMahsaAmini/Core#468] [Q&A]
Dear Farnood,

I am representing a community of information technology professionals who wanted to take a very similar initiative to yours and create an anonymized, open-source, non-profit organization/community to scale up and empower the volunteers who are fed up and want to do something about the current situation in Iran.

We do not want to reinvent the solution you are already providing, and it's in everyone's best interest to stay focused within the same community. Therefore, on behalf of our community, I would like to address a few concerns we came across while performing due diligence on your initiative. We would appreciate it if you could help us understand your thoughts on each matter.

* Our first and most significant concern is that you are exposing your identity, which shows your initiative's strength, support, and seriousness while adding a significant risk for you and your family being targeted by IRGC/MIOS. 
  * What are the contingency plans for the organization in case of your arrest or unavailability due to any reasons?
* Would you please describe what your current decision-making process looks likes? 
  * For instance, we all agree with your current chants and goals, but if in the future you decide to update them while we are contributing to your organization, how would you do it?
* How can you ensure the integrity of developed goals, content, or technologies?
  * For instance, how do you ensure you can't upload any malicious code to the repositories in case you or your system is compromised?

I will address more concerns once we hear your response to the current ones, and we look forward to hearing from you.

Bests,
The auditor.

**Comment by Pink3lephant** [comment]
Hi! Thanks for the great list of questions. 

I think for any successful organisation, having answer for these questions is important.

Can you please reach out to me at freeinternettech@gmail.com to discuss it further? We are building a similar community and would love to join forces 💪 

Link to the website: https://freeinternet.app/

**Comment by Amini-Community-Auditor-GitHub** [comment]
Hi @Pink3lephant,

Thank you for your invitation. Let's keep our discussion public for clarification. So please provide your answers here.

@farnoodma, we are looking forward to hearing from you. Please let me know if you need more information to provide an answer to our concerns.

Bests,
The auditor.

**Comment by kiana-k7597** [comment]
Also would be interested in the answers to these questions.

**Comment by tcf-admin** [comment]
@Amini-Community-Auditor-GitHub, 
We would like to introduce TCF e.V.

- We are a heterarchical NG-NPO civic lab registered under VR:725308 in Germany.
- We make decisions sociocratically & based on consent. 
- Our framework is based on human rights and fundamental freedoms.
- We take security and safety of our team members seriously and working with a pseudonym, through an advocate, and as an organisation is possible within TCF e.V..
- We have both internal and external audits for every one of our public-facing products.
- We are open and produce open source, locks and keys are held by a group of fully-identified and committed individuals and responsibility is shared in a way that physically removing people would not stop the routines. 

You want to know more about our decision-making procedures, standards and members?
Write us a letter and we invite you to our general assembly convention (17.12.2022)
info transcf org
